### DISCUSSION TITLE
Web3auth login in-app browser flow ends with Torus support chatbot

### DISCUSSION THREAD

**Discussion by Haris565** [Web3Auth/Web3Auth#746] [Help]
Hi

I'm using the React Native SDK to login in with a social login provider (Apple / Google) some time signin flow works smooth and sometime it redirects the user to torus helpdesk i tried everything the issue still persist can anyone guide me how to disable this helpdesk or is there any way out

If an error caused this behaviour, can it close the in-app browser and throw an error, rather than redirecting to this page?

Is there a fix for this? Maybe a parameter that I can pass into the initialisation object?

Thank you.

![helpdesk](https://user-images.githubusercontent.com/64888353/196910152-2cdf88c7-f325-4a82-8844-28bd53a24654.jpeg)

**Comment by BboyStatix** [comment]
Hi there @Haris565 could you please provide some logs to help us debug
